### PR TITLE
Updating README

### PR DESCRIPTION
**Pull request by kenodegard**
### Description

1. Removed alternative install instructions as we don't want to encourage modifying conda source code for plugins to work.
2. Concatenate paragraphs for consistency.
3. Minor edits
    - added link to this template
    - updated folder name

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -3,14 +3,14 @@ requires = ["setuptools>=61.0", "setuptools-scm"]
...
 
 [project]
-name = "my-conda-subcommand"
-version = "1.0.0"
+name = "string-art"
+version = "1.0"
 description = "My string art subcommand plugin"
 requires-python = ">=3.7"
-dependencies = ["conda", "art"]
+dependencies = ["conda", "pyfiglet"]
```
Comment (beeankha) [inline pyproject.toml:10]: Could we please also change the [subcommand module](https://github.com/conda/conda-plugin-template/blob/main/string_art.py) to import `pyfiglet` vs `art`?

Comment (kenodegard) [reply pyproject.toml:10]: ah I had been planning to do that as a separate PR (this accidentally slipped through), let me revert this to `art` and then we can do that separately

File: README.md [hunk 2]
```diff
@@ -19,32 +18,28 @@ $ conda activate plugin-tutorial
 
 ## Project directory structure
 
-Set up your working directory and files as shown below:
+Set up your working directory and files as shown below (or create a new repo using this [template][https://github.com/conda/conda-plugin-template/generate]):
```
Comment (beeankha) [inline README.md:21]:
```suggestion
Set up your working directory and files as shown below (or create a new repo using this [template][template]):
```

**Review by beeankha — DISMISSED** [review]
Other than the one link fix, everything looks good to me! 👍

**Review by beeankha — APPROVED** [review]

**Pull request merged**